### DISCUSSION TITLE
Mortgage Layout

### DISCUSSION THREAD

**Discussion by MahbobaAriai** [Sudeep-Magar/Tafe240260#7] [General]
Hey Sudeep 
Just wanted to know what do you think about the Mortgage Calculator Layout.
Let me know if I need to change anything.
Thanks 
Mahboba
![MortgageCalc](https://github.com/user-attachments/assets/c2daba1f-75e0-47c2-a03f-a4b4de86dd34)

**Comment by Sudeep-Magar** [comment]
looks good to me.
